PYME/Analysis/points/traveling_salesperson/two_opt_utils.c

```diff
@@ -0,0 +1,8019 @@
+/* Generated by Cython 0.29.13 */
```

Comment:
@b3nroll1ns I was confused earlier - I hadn't actually added this to the repository but see now that you are with this PR. I'm guessing this was an accident since you didn't change the pyx file (and to my knowledge weren't working on this part of the code). The pyx has everything to recompile and we don't want this entire file changing every-time someone does a development install. I'll submit a PR to add this to gitignore to make it easier to avoid in the future.

Reply:
Another way to avoid this is to only add certain files to the commit. For example, if you change `kinetics_recipes.py`, you can type `git add kinetics_recipies.py`. If you do this individually for each file you modify, you'll never end up adding other files that get generated by Cython, caching, etc. to the commit. Alternatively, if there really is only one extraneous file, you can type `git add .` to add all the files in the folder to the commit and then `git reset -- two_opt_utils.c` to remove the unwanted file from the commit.

Reply:
Thanks for the advice, I'll do this going forward!